virt_lightning/__init__.py

```diff
@@ -187,6 +187,21 @@ def start(self, domain):
         domain.attachDisk(meta_data_iso, device="cdrom", disk_type="raw")
         domain.dom.create()
 
+    def clean_up(self, domain):
+        xml = domain.dom.XMLDesc(0)
+        root = ET.fromstring(xml)
+        for disk in root.findall("./devices/disk[@type='file']/source[@file]"):
+            filepath = disk.attrib["file"]
+            try:
+                self.conn.storageVolLookupByPath(filepath).delete()
+            except libvirt.libvirtError as e:
+                if e.get_error_code() == libvirt.VIR_ERR_NO_STORAGE_VOL:
+                    pass
```

Comment:
Useful handle all exception and show in logs.

Reply:
Agreed, but we don't have a logging system yet. I don't want to flood the console with low important messages. I just added this card: https://github.com/virt-lightning/virt-lightning/issues/32